### PR TITLE
Updated readme

### PR DESCRIPTION
**Pull request by nnvn**
Updated [internal/docs/android.md](internal/docs/android.md) and added link to it in readme.md

### CODE REVIEW

**Comment by m8m5** [comment]
please replace `(internal/examples/androidextras/jni)` with `(../examples/androidextras/jni)` in `android.md`. 
current link lead to a 404 page.

File: internal/docs/android.md [hunk 1]
```diff
@@ -42,3 +42,9 @@ identifier, icon, permissions, ...)
 
 For example: `android:icon="@drawable/icon"` attribute can be added to
 `application` in `AndroidManifest.xml` to use the `icon.png`.
+
+## Java native interface
+
+You can call android's java runtime directly from go. See [java native interface example](internal/examples/androidextras/jni)
```
Comment (m8m5) [inline internal/docs/android.md]: please replace `(internal/examples/androidextras/jni)` with `(../examples/androidextras/jni)`.

current link leads to a 404 page.

Comment (nnvn) [inline internal/docs/android.md]: please check now

Comment (m8m5) [inline internal/docs/android.md]: looks good 👍

**Comment by therecipe** [comment]
Thank you both :)

**Pull request merged**